app/views/results.nunjucks

```diff
@@ -18,14 +18,29 @@
 {% endblock %}
 
 {% block content %}
-<div class="reading-width">
+<div class="reading-width results">
   {% if gps.length === 0 %}
     {% include "includes/search-form.nunjucks" %}
   {% else %}
     <h1 class="sr-only" data-results="{{gps.length}}" data-term="{{search}}">Search results</h1>
     <h2 class='results__header'>{{resultsHeader}}</h2>
-    <p>{{searchInformationMessage}}<p>
-    <p>{{searchHelpMessage | safe }}<p>
+    {% if (search) and (postcode) %}
```

Comment:
@neilbmclaughlin @ajthomascouk @ermlikeyeah  - I seem to accidentally merged this in this morning and it propagated to the other stuff :| hence why the messaging was off. Sending a PR now.